TRACONBoundaries.geojson

```diff
@@ -468,7 +468,21 @@
     {"type":"Feature","properties":{"id":"UMOO","prefix":["UMOO_TWR"],"name":"Homiel Tower"},"geometry":{"type":"MultiPolygon","coordinates":[[[[30.61805556,54.17000000],[30.83999972,53.86416667],[30.75805583,53.64500000],[30.43305583,53.55166667],[30.14972167,53.49166694],[29.76055583,53.61694389],[29.38972194,53.81833222],[29.33305556,53.97805583],[29.44138861,54.18333278],[29.50777778,54.25416667],[29.99305556,54.21861083],[30.23000000,54.16111028]]]]}},
-    {"type":"Feature","properties":{"id":"URSS","prefix":["URSS"],"name":"Sochi Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[38.703889,44.310556],[38.965,44.333056],[39.448333,44.366389],[39.84,44.369167],[39.955,44.381944],[40.027778,43.954167],[40.533333,43.95],[40.710278,43.881389],[41.148333,43.403333],[41.148333,43.403333],[41.122722,43.398603],[41.103467,43.403140],[41.103896,43.394631],[41.096364,43.386224],[41.079293,43.390743],[41.073238,43.387732],[41.005646,43.431611],[40.995579,43.432348],[40.991460,43.431246],[40.985991,43.430962],[40.983237,43.430135],[40.982619,43.427819],[40.979107,43.425511],[40.957683,43.420087],[40.952083,43.419357],[40.944174,43.425568],[40.930031,43.426569],[40.926267,43.434256],[40.929474,43.444389],[40.892706,43.457935],[40.893004,43.459971],[40.899433,43.468465],[40.898309,43.476197],[40.896806,43.478684],[40.891347,43.482247],[40.894408,43.487379],[40.890591,43.489634],[40.837054,43.489500],[40.829646,43.488100],[40.818992,43.492332],[40.755928,43.519745],[40.745200,43.521425],[40.740898,43.526563],[40.736724,43.528395],[40.693357,43.535176],[40.688624,43.540796],[40.692434,43.547910],[40.676009,43.551317],[40.674459,43.557809],[40.671911,43.557971],[40.661813,43.563127],[40.648522,43.555642],[40.627446,43.549068],[40.620460,43.551093],[40.565375,43.518772],[40.559729,43.520029],[40.517496,43.524529],[40.475449,43.550068],[40.432276,43.558095],[40.427161,43.553557],[40.249021,43.586395],[40.211523,43.575245],[40.106611,43.570072],[40.090640,43.555230],[40.094974,43.530057],[40.081413,43.519188],[40.040266,43.452517],[40.012964,43.430154],[40.010240,43.423420],[40.015349,43.420601],[40.006807,43.386875],[40.011667,43.383333],[39.916667,43.188333],[40.075,42.7],[39.621667,42.605278],[39.200278,42.696111],[38.2,43.041667],[38.171944,43.086389],[38.083333,43.513333],[38.281667,43.898056],[38.405,44.038056],[38.703889,44.310556]]]]}},
+	{"type":"Feature","properties":{"id":"URKG","prefix":["URKG"],"suffix":"R_APP","name":"Gelendzhik Tower"},"geometry":{"type":"MultiPolygon","coordinates":[[[[38.185,44.725],[38.353333333333,44.585],[38.408333333333,44.366388888889],[38.703888888889,44.310555555556],[38.405,44.038055555556],[38.281666666667,43.898055555556],[38.236666666667,43.81],[37.329722222222,44.441111111111],[37.403611111111,44.479722222222],[37.561666666667,44.631388888889],[37.670833333333,44.763055555556],[38.185,44.725]]]]}},
```

Comment:
This project is in general for Approach/Departure/TRACON positions and not for tower. Is this actual tower airspace or is it a tower operating as approach? If it is just tower airspace, it should be removed.

Reply:
It is  tower operating as approach. Tower is just callsign.